Assets/Scripts/UI/CombatHUD/CombatTutorial.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CombatTutorial : MonoBehaviour
+{
+	private bool tutorialOn;
+	private void Awake()
+	{
+		tutorialOn = true;
+	}
+
+	private float timer = 0.0f;
+	public float secondsToDisplay;
```

Comment:
Tiny thing, but I think public variables should be PascalCase rather than camelCase. c:

Reply:
Whoops!